src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -901,12 +897,8 @@ private static String replaceUnpredictableValuesWithPlaceholders(String content)
                 // today's date
                 .replace(TimeHelper.formatDate(now).replace("/", "&#x2f;"), "${today}")
                 .replace(TimeHelper.formatDate(now), "${today}")
-                // now (used in comments last edited date) e.g. [Thu, 07 May 2015, 07:52:13 UTC]
-                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC", "\\${comment\\.date}")
-                // now (date, time)
-                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2} [AP]M", "\\${datetime\\.now}")
-                // dynamic feedback submission numbers
-                .replaceAll("(?s)<span class=\"submissionsNumber\".*?</span>", "<span class=\"submissionsNumber\" id=\"submissionsNumber\">\\${submissions\\.number}</span>")
+                // now (date, time) e.g. [Thu, 07 May 2015, 07:52 PM] or [Thu, 07 May 2015, 07:52:13 UTC]
+                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}( [AP]M|:[0-9]{2} UTC)", "\\${datetime\\.now}")
```

Comment:
Yes I understand what you're trying to say, but let me remind you of a similar case: the standardized truncated accounts. We agreed to suppress truncated accounts back to its original when we could have used `${test.truncated.account}`s, because what's the point? More placeholders means more `.replace(...)` which makes the comparison slower, and the only advantage is we know that the accounts are truncated in places where they should be truncated, which is easily achieved by hard-coding _just one_ test case to confirm it.

This standardization of placeholders is no different. Only that there is not just one hard-coded comment dates in test files, but hundreds.

And I do realize that combining the regexes (`( [AP]M|:[0-9]{2} UTC)`) is not that much faster compared to the two separate replaces, which is why I attempted to standardize also the date/time format. My reason for removing the placeholders should have been obvious: for performance purpose.